plugins/woocommerce/src/Internal/Admin/Settings/PaymentProviders/WooPayments/WooPaymentsService.php

```diff
@@ -296,38 +142,93 @@ private function get_onboarding_steps_details( string $location, string $rest_pa
 	 * @return string The status of the onboarding step.
 	 */
 	public function get_onboarding_step_status( string $step_id, string $location ): string {
```

Comment:
Added "sophistications" around the WPCOM connection, test account, and business verification logic.